### PR TITLE
console: new set of screenshots for the Renaming Grande

### PR DESCRIPTION
**Pull request by ultraschuppi**
## Description

for https://github.com/camunda-cloud/camunda-cloud-management-apps/issues/1200 / https://github.com/camunda/product-hub/issues/1560

## When should this change go live?

changes are already live

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [X] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by ultraschuppi** [comment]
@christinaausley it is about time again ;-) a 2nd eye pair checking the generated screenshots would be very much appreciated!

changes are both on next and 8.2

File: docs/components/console/manage-clusters/img/cluster-details-created-client.png
Comment (christinaausley) [inline docs/components/console/manage-clusters/img/cluster-details-created-client.png:1]: This is now "APIs & tools" 👍

File: docs/components/console/manage-organization/img/delete-account.png
Comment (christinaausley) [inline docs/components/console/manage-organization/img/delete-account.png:1]: Same here for PII.

File: versioned_docs/version-8.2/components/console/manage-plan/img/checkout.png
Comment (christinaausley) [inline versioned_docs/version-8.2/components/console/manage-plan/img/checkout.png:1]: Professional is changing to Starter, but I'm assuming this will be corrected in a separate PR? CC @jprcoliveira

**Review by christinaausley — COMMENTED** [review]
A few non-blocking comments.

Additionally, is it possible to set up a separate issue to use sentence-case formatting throughout our Console? I'm happy to tackle this at another time if you want to show me how to do it 😄

File: docs/components/console/manage-organization/img/activity-view.png
[the remark this answers is not in the document]
Comment (ultraschuppi) [reply docs/components/console/manage-organization/img/activity-view.png:1]: i don't care ;)

File: versioned_docs/version-8.2/components/console/manage-plan/img/checkout.png
Comment (ultraschuppi) [reply versioned_docs/version-8.2/components/console/manage-plan/img/checkout.png:1]: the screenshots are taken from prod - so as long as we do not have the new stuff on prod, the screenshots will not display the starter plan.

i am fine with running another screenshot-automation-run once the starter thing has launched - but yes: @jprcoliveira you will take care of the docs-wording-changes for the upcoming starter plan?

File: docs/components/console/manage-clusters/img/cluster-details-created-client.png
Comment (ultraschuppi) [reply docs/components/console/manage-clusters/img/cluster-details-created-client.png:1]: what do you mean @christinaausley ?

**Comment by ultraschuppi** [comment]
> A few non-blocking comments.
> 
> Additionally, is it possible to set up a separate issue to use sentence-case formatting throughout our Console? I'm happy to tackle this at another time if you want to show me how to do it 😄

i get my designs **and wording** from @karl-heinsen - so probably best to sync with him. i am fine if you want to do that in console-codebase, but actually also not a biggie for myself.

**Comment by karl-heinsen** [comment]
> Additionally, is it possible to set up a separate issue to use sentence-case formatting throughout our Console? I'm happy to tackle this at another time if you want to show me how to do it 😄
 
@christinaausley, I'm glad you brought this up. Updating the older title case text to sentence case has been on my to-do list for a while. I have been updating the formatting when I work on a specific feature, but it would be great to approach this under one main issue. In that issue, it would also be great to tackle what needs to be capitalized and not. For now, we follow Carbon's guide - _Do not capitalize the names of features and components unless they are sold separately or are trademarked._

File: docs/components/console/manage-clusters/img/cluster-details-created-client.png
Comment (christinaausley) [reply docs/components/console/manage-clusters/img/cluster-details-created-client.png:1]: The document is now called "Working with APIs and tools" rather than "Working with APIs and clients" as we have here 👍

**Comment by akeller** [comment]
> For now, we follow Carbon's guide - _Do not capitalize the names of features and components unless they are sold separately or are trademarked._

@christinaausley @karl-heinsen I would recommend adding what formatting or guidance we use in the product UI to internal and/or external style guide assets.

**Comment by christinaausley** [comment]
> @christinaausley, I'm glad you brought this up. Updating the older title case text to sentence case has been on my to-do list for a while. I have been updating the formatting when I work on a specific feature, but it would be great to approach this under one main issue. In that issue, it would also be great to tackle what needs to be capitalized and not. For now, we follow Carbon's guide - _Do not capitalize the names of features and components unless they are sold separately or are trademarked._

@karl-heinsen Are you comfortable opening an issue for this? Currently, we do uppercase components, for example:
"How to use Optimize" rather than "How To Use Optimize" or "How to use optimize" 👍
